api/src/main/java/uk/gov/hmcts/payment/api/controllers/LegacyReferenceController.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.payment.api.controllers;
+
+
+import io.swagger.annotations.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.payment.api.dto.LegacySiteDTO;
+import uk.gov.hmcts.payment.api.model.LegacySite;
+import uk.gov.hmcts.payment.api.service.LegacySiteService;
+
+import java.util.List;
+
+/**
```

Comment:
remove all comments, author, version, sinse annotations.

Reply:
Removed All the comments